startFromVcf.sh

```diff
@@ -2,7 +2,6 @@
 
 set -e
 set -u
-module load NGS_DNA/3.5.2
 module list
```

Comment:
Replacing the hard coded version number for a variable which comes from a config file or commandline option makes sense to me, but completely removing the `module load` statement makes less sense to me.

Reply:
In the commandline help this is listed:
```
-a   sampleType (DNA or RNA) (default=DNA)
```
but there are several instances of `${EBROOTNGS_DNA}` which will not work for RNA data, right? You could also check if `${EBROOTNGS_DNA}` is defined and if not or empty, exit with an error message that a `module load NGS_DNA` is required.